Updating people and projects for OLS-2 (#190)
open-life-science/open-life-science.github.io: ols-2.md

Participants join this program with a project that they either are already working on or want to develop during this program. More details about the role of a project lead (mentee) can be found [here](/about#mentees).

> For the second round of the Open Life Science program, we are happy to have [{{ p-participants | size }} participants](/ols-2/projects-participants#participants) with [{{ projects | size }} projects](/ols-2/projects-participants#projects). 

Author: ```suggestion
For the second round of the Open Life Science program, we welcome [{{ p-participants | size }} participants](/ols-2/projects-participants#participants) with [{{ projects | size }} projects](/ols-2/projects-participants#projects). 
```
